config.py

```diff
@@ -20,7 +20,13 @@ def __getattr__(self, attrname):
             warnings.warn("The `slack_name` key in %s is deprecated in favor of the `SLACK_NAME` environment variable" %
                           self.config_fname, DeprecationWarning)
 
-        return self.config[attrname]
+        if os.getenv(attrname.upper()) is not None:
+            return os.getenv(attrname.upper()).split(',') if ',' in os.getenv(attrname.upper()) else os.getenv(attrname.upper())
+
+        try:
```

Comment:
What was the exception here? KeyError? If so, `return self.config.get(attrname, '')` is a little cleaner.